public/styles.css

```diff
@@ -1,6 +1,6 @@
 /* This file is for global styles only */
 :root {
-  --game-window: grey;
+  --game-window: rgb(134, 134, 134);
```

Comment:
looks good, love that you are making variables here in case we ever decide to change colors. If you could, would you make a css folder in the path and drop this file into there so we can keep all css together.